js/app.jsx

```diff
@@ -517,6 +537,8 @@ var Job = React.createClass({
       ['State', state],
       ['Input', inputs],
       ['Output', cleanJobPath(job.conf.output)],
+      ['Pipe', pipe],
+      ['Step', step],
```

Comment:
I don't understand what `Pipe` or `Step` mean here. We should figure out more obvious terminology. I think one of them one of them maps to the input and output paths but it's hard to tell.